app/views/price_groups/index.html.haml

```diff
@@ -1,4 +1,4 @@
-x= content_for :h1 do
+= content_for :h1 do
   = current_facility
 = content_for :sidebar do
   = render :partial => 'admin/shared/sidenav_admin', :locals => { :sidenav_tab => 'price_groups' }
```

Comment:
Let's get some double quotes and updated hash syntax in the next pass.

Reply:
👍